source/_patterns/04-components/mobile-menu/_mobile-menu.scss

```diff
@@ -3,22 +3,18 @@
 
 $mobile-menu-fallback-bg-color: #f3f3f3 !default;
```

Comment:
Colors defined at the top of this file should be changed to follow other components, using colors from the 'colors' section of the design tokens file.